pkg/authenticator/authenticator.go

```diff
@@ -133,7 +152,35 @@ func (auth *Authenticator) Login() error {
 		return log.RecordedError(log.CAKC029E, err.Error())
 	}
 
-	// load client cert
+	return nil
+}
+
+// Returns true if we are logged in (have a cert)
+func (auth *Authenticator) IsLoggedIn() bool {
+	return auth.PublicCert != nil
+}
+
+// CertFileExist returns true if certificate file exists in the ClientCertPath
+func (auth *Authenticator) CertFileExist() bool {
+	_, err := os.Stat(auth.Config.ClientCertPath)
+	return !os.IsNotExist(err)
+}
+
+// Returns true if certificate is expired or close to expiring
+func (auth *Authenticator) IsCertExpired() bool {
+	certExpiresOn := auth.PublicCert.NotAfter.UTC()
+	currentDate := time.Now().UTC()
+
+	log.InfoLogger.Printf(log.CAKC008I, certExpiresOn)
+	log.InfoLogger.Printf(log.CAKC009I, currentDate)
+	log.InfoLogger.Printf(log.CAKC010I, bufferTime)
```

Comment:
why are these being logged? I already have concerns about the number of log entries in this project since the values are also logged in secretless and it creates a LOT of extra log messages - here we add three, that look more like they should be debug than info level

Reply:
edit to add: I see that this is how we did this below, but my concern about these being debug and not info messages stands